recipes_source/torch_compile_backend_ipex.rst

```diff
@@ -0,0 +1,157 @@
+Intel® Extension for PyTorch* Backend
+=====================================
+
+To work better with `torch.compile`, Intel® Extension for PyTorch* implements a backend `ipex`. It targets to improve hardware resource usage efficiency on Intel platforms for better performance. The `ipex` backend is implemented with further customizations designed in Intel® Extension for PyTorch* for the model compilation.
+
+Usage Example
+~~~~~~~~~~~~~
+
+Train FP32
+----------
+
```

Comment:
Can we add something like: "The following example demonstrates...." and describe what it demonstrates.

Reply:
Sure. Added description below.